integration-libs/cdc/user-account/login-form/reconsent/cdc-reconsent.service.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, OnDestroy } from '@angular/core';
```

Comment:
Check if the whole service can be moved to one service where we have everything for the Consent mgmt feature.